### PR TITLE
UI test branch

### PR DESCRIPTION
**Pull request by tlpackus**
We added confirmation.html to display the links on a seperate page updated the index.html with selector boxes and are in the process of updating main.js

### CODE REVIEW

**Review by JuanHasbunZem — APPROVED** [review]

File: src/main.js [hunk 1]
```diff
@@ -2,24 +2,37 @@ import $ from 'jquery';
 import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './css/styles.css';
-import { Catalog } from './js/course-catalog.js';
+// import { Catalog } from './course-catalog.js';
```
Comment (shanole) [inline src/main.js]: Do y'all need the commented out code? If not, can we delete it from the repo file so that we minimize any potential merge conflicts?

Comment (shanole) [reply src/main.js:6]: also the console.logs unless you still need them!!

Comment (shanole) [reply src/main.js]: also the console.logs unless you still need them!!

**Comment by tlpackus** [comment]
just took out the comments

**Review by jessicarubinwilliams — APPROVED** [review]

**Review by shanole — APPROVED** [review]

**Pull request merged**